src/frontend/komponenter/Oppgavebenk/OppgaveDirektelenke.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from 'react';
+
+import { useHistory } from 'react-router';
+
+import { RessursStatus } from '@navikt/familie-typer';
+
+import { useApp } from '../../context/AppContext';
+import { useFagsakRessurser } from '../../context/FagsakContext';
+import { useOppgaver } from '../../context/OppgaverContext';
+import { IOppgave, oppgaveTypeFilter, OppgavetypeFilter } from '../../typer/oppgave';
+import { hentFnrFraOppgaveIdenter } from '../../utils/oppgave';
+import FamilieBaseKnapp from '../Felleskomponenter/FamilieBaseKnapp';
+
+interface IOppgaveDirektelenke {
+    oppgave: IOppgave;
+}
+
+const OppgaveDirektelenke: React.FC<IOppgaveDirektelenke> = ({ oppgave }) => {
+    const { harLøpendeSakIInfotrygd, sideindeks } = useOppgaver();
+    const { hentFagsakForPerson } = useFagsakRessurser();
+    const { sjekkTilgang } = useApp();
+    const [feilmelding, settFeilmelding] = useState<string>('');
+    const history = useHistory();
+    const oppgavetype = oppgaveTypeFilter[oppgave.oppgavetype]?.id;
+
+    useEffect(() => {
+        settFeilmelding('');
+    }, [sideindeks]);
+
+    const visTilgangsmodalEllerSendVidere = async (oppgave: IOppgave) => {
+        const brukerident = hentFnrFraOppgaveIdenter(oppgave.identer);
+
+        if (brukerident) {
+            if (await sjekkTilgang(brukerident)) {
+                const løpendeSak = await harLøpendeSakIInfotrygd(brukerident);
+                if (løpendeSak.status === RessursStatus.SUKSESS) {
+                    if (løpendeSak.data.harLøpendeSak) {
+                        history.push('/infotrygd', { bruker: brukerident });
+                    } else {
+                        history.push(`/oppgaver/journalfør/${oppgave.id}`);
+                    }
+                }
+            }
+        } else {
+            history.push(`/oppgaver/journalfør/${oppgave.id}`);
+        }
+    };
+
+    const sjekkTilgangOgGåTilBehandling = async (oppgave: IOppgave) => {
```

Comment:
Burde vi ha spinner eller lignende på knappen istedet for "systemet laster"?

Reply:
siden forventet flyt er ny side tenker jeg "systemet laster" er ok egentlig. Det er mest en sikring hvis noen har oppretta oppgave kun i gosys f.eks, i overgangsfase

Reply:
Ja, mulig, men ville hørt med fag. For meg virket det som at systemet hang når meldingen ble borte, for så å oppdatere siden litt senere.

Reply:
oppdatert video nå :)